src/main/webapp/app/entities/ace-editor/session.model.ts

```diff
@@ -0,0 +1,3 @@
+import { AceAnnotation } from './annotation.model';
+
+export type Session = { ts: number; errors: { [fileName: string]: AceAnnotation[] } };
```

Comment:
would it make sense to use a proper class type here?

Reply:
I think in this case Session is really just a plain object without any functionality attached to it. So I would suggest to keep it lightweight in this case.